src/VolleyManagement.Backend/VolleyManagement.Domain/PlayersAggregate/Player.cs

```diff
@@ -102,7 +101,7 @@ public string LastName
             {
                 if (PlayerValidation.ValidateHeight(value))
                 {
-                    throw new ArgumentException(Resources.ValidationPlayerHeight, "Height");
+                    throw new ArgumentException(Resources.ValidationPlayerHeight, nameof(Player.Height));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The parameter name 'Height' is not declared in the argument list. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3928)